CONTRIBUTING.rst

```diff
@@ -138,25 +139,26 @@ Ready to contribute? Here's how to set up `renku` for local development.
 Commit message guidelines
 -------------------------
 
-This project is using conventional commits style for generation of changelog upon each release. Therefore,
-it's important that our commit messages convey what they do correctly. Commit message should always follow this pattern:
+This project is using conventional commits style for generation of changelog upon
```

Comment:
Maybe add a link to conventional commits?

Reply:
Added in #4fd255